pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -134,9 +132,11 @@ def __init__(self, filepath: Optional[str] = None, monitor: str = 'val_loss', ve
         self.verbose = verbose
         if filepath is None:  # will be determined by trainer at runtime
             self.dirpath, self.filename = None, None
+        elif filepath == "":  # uses CWD as the dirpath
+            self.dirpath, self.filename = os.path.realpath(filepath), None
```

Comment:
```suggestion
        if not filepath:  # will be determined by trainer at runtime
            self.dirpath, self.filename = None, None
```
should we do this?

Reply:
Discussed here: https://github.com/PyTorchLightning/pytorch-lightning/pull/3163#discussion_r483197256

Reply:
`filepath=""` is just an empty string equivalent to None. I prefer `"" the same as None, and have "." and "./" lead to cwd` as @awaelchli said and maybe update the test if they are failing.

Reply:
Done.